compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs

```diff
@@ -810,15 +808,36 @@ renderValue world name = \case
         renderField (Field label mbValue) =
             renderValue world (name ++ [TL.toStrict label]) (fromJust mbValue)
 
+templateConName :: LF.World -> Identifier -> Maybe (LF.Qualified LF.TypeConName)
+templateConName world (Identifier mbPkgId (TL.toStrict -> qualName)) = do
+  (modName, defName, mod0) <- lookupModuleFromQualifiedName world mbPkgId qualName
+  tpl <- NM.lookup (Tagged [defName]) (LF.moduleTemplates mod0)
```

Comment:
Using `Tagged` is an anti pattern. Is there a `mkModuleName` or similar that wraps that?